webpack.config.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Left here in case esbuild stops working for us when we try to re-enable
```

Comment:
This comment is good, but this should be renamed `webpack.config.js.example` or all the code in this file should be commented out if we're not actually using it. It's likely a future developer will land in the middle of this file by searching or just miss the comment and get really confused about how the module is built. It might also be good to name check the actual build system (esbuild?) in this comment.